src/SenseNet.Core.Tests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
```

Comment:
The whole SenseNet.Core.Tests folder can be deleted, as the tests were moved to a different project.